src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.builds

```diff
@@ -2,21 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="runtime.native.System.IO.Compression.pkgproj"/>
```

Comment:
That's intentional.  They are just markers that can be referenced by other non-distro-specific package.  The runtime.json in the lineup package is what brings in the platform specific implementation package.